common/common/src/main/kotlin/com/cosmotech/api/security/AbstractSecurityConfiguration.kt

```diff
@@ -2,51 +2,44 @@
 import org.springframework.security.oauth2.jwt.Jwt
 import org.springframework.security.oauth2.jwt.JwtClaimValidator
-import org.springframework.security.oauth2.jwt.JwtDecoder
-import org.springframework.security.oauth2.jwt.NimbusJwtDecoder
 import org.springframework.web.cors.CorsConfiguration
 
 // Business roles
-const val ROLE_PLATFORM_ADMIN = "APPROLE_Platform.Admin"
-const val ROLE_CONNECTOR_DEVELOPER = "APPROLE_Connector.Developer"
-const val ROLE_ORGANIZATION_ADMIN = "APPROLE_Organization.Admin"
-const val ROLE_ORGANIZATION_COLLABORATOR = "APPROLE_Organization.Collaborator"
-const val ROLE_ORGANIZATION_MODELER = "APPROLE_Organization.Modeler"
-const val ROLE_ORGANIZATION_USER = "APPROLE_Organization.User"
-const val ROLE_ORGANIZATION_VIEWER = "APPROLE_Organization.Viewer"
+const val ROLE_PLATFORM_ADMIN = "Platform.Admin"
+const val ROLE_CONNECTOR_DEVELOPER = "Connector.Developer"
+const val ROLE_ORGANIZATION_ADMIN = "Organization.Admin"
+const val ROLE_ORGANIZATION_COLLABORATOR = "Organization.Collaborator"
+const val ROLE_ORGANIZATION_MODELER = "Organization.Modeler"
+const val ROLE_ORGANIZATION_USER = "Organization.User"
+const val ROLE_ORGANIZATION_VIEWER = "Organization.Viewer"
 
 // Endpoints roles
-const val ROLE_CONNECTOR_READER = "APPROLE_Connector.Reader"
-const val ROLE_CONNECTOR_WRITER = "APPROLE_Connector.Writer"
-const val ROLE_DATASET_READER = "APPROLE_Dataset.Reader"
-const val ROLE_DATASET_WRITER = "APPROLE_Dataset.Writer"
-const val ROLE_ORGANIZATION_READER = "APPROLE_Organization.Reader"
-const val ROLE_ORGANIZATION_WRITER = "APPROLE_Organization.Writer"
-const val ROLE_SCENARIO_READER = "APPROLE_Scenario.Reader"
-const val ROLE_SCENARIO_WRITER = "APPROLE_Scenario.Writer"
-const val ROLE_SCENARIORUN_READER = "APPROLE_ScenarioRun.Reader"
-const val ROLE_SCENARIORUN_WRITER = "APPROLE_ScenarioRun.Writer"
-const val ROLE_SOLUTION_READER = "APPROLE_Solution.Reader"
-const val ROLE_SOLUTION_WRITER = "APPROLE_Solution.Writer"
-const val ROLE_WORKSPACE_READER = "APPROLE_Workspace.Reader"
-const val ROLE_WORKSPACE_WRITER = "APPROLE_Workspace.Writer"
+const val ROLE_CONNECTOR_READER = "Connector.Reader"
+const val ROLE_CONNECTOR_WRITER = "Connector.Writer"
+const val ROLE_DATASET_READER = "Dataset.Reader"
+const val ROLE_DATASET_WRITER = "Dataset.Writer"
+const val ROLE_ORGANIZATION_READER = "Organization.Reader"
+const val ROLE_ORGANIZATION_WRITER = "Organization.Writer"
+const val ROLE_SCENARIO_READER = "Scenario.Reader"
+const val ROLE_SCENARIO_WRITER = "Scenario.Writer"
+const val ROLE_SCENARIORUN_READER = "ScenarioRun.Reader"
+const val ROLE_SCENARIORUN_WRITER = "ScenarioRun.Writer"
+const val ROLE_SOLUTION_READER = "Solution.Reader"
+const val ROLE_SOLUTION_WRITER = "Solution.Writer"
+const val ROLE_WORKSPACE_READER = "Workspace.Reader"
+const val ROLE_WORKSPACE_WRITER = "Workspace.Writer"
+
+// Allowed scopes
+const val SCOPE_SCENARIO_READ = "SCOPE_csm.scenario.read"
```

Comment:
Could be nice to do the others...
Connector, dataset, organization, scenariorun, solution & workspace.
For Read and Write.

Reply:
done